app/views/issues/_attributes.html.erb

```diff
@@ -1078,6 +1078,28 @@ $(document).ready(function() {
     $da.on('change', updateFieldVisits)
   );
 
+  function convertTo24HourFormat(dateTimeString) {
+    if (!dateTimeString) return dateTimeString;
+
+    if (!/AM|PM/i.test(dateTimeString)) return dateTimeString;
+
+    const match = dateTimeString.match(/^(\d{4}-\d{2}-\d{2})\s+(\d{1,2}):(\d{2})\s*(AM|PM)$/i);
+    if (!match) return dateTimeString;
+
+    const [, datePart, hourStr, minuteStr, ampm] = match;
+    let hour = parseInt(hourStr, 10);
+    const minute = minuteStr;
+
+    if (ampm.toUpperCase() === 'AM') {
+      if (hour === 12) hour = 0;
+    } else { // PM
+      if (hour !== 12) hour += 12;
+    }
+
+    const hourPadded = hour.toString().padStart(2, '0');
+    return `${datePart} ${hourPadded}:${minute}`;
+  }
+
```

Comment:
This looks like it will work, but it might be better to let the `Date` class handle the formatting. Here's an example:
```js
function convertTo24HourFormat(dateTimeString) {
  if (!dateTimeString) return dateTimeString;

  const d = new Date(dateTimeString);
  if (isNaN(d)) return dateTimeString;

  return new Intl.DateTimeFormat("en-CA", {
    year: "numeric",
    month: "2-digit",
    day: "2-digit",
    hour: "2-digit",
    minute: "2-digit",
    hour12: false
  }).format(d).replace(",", "");
}
```